src/main/kotlin/net/maizegenetics/phgv2/cli/CreateRanges.kt

```diff
@@ -45,6 +51,12 @@ class CreateRanges: CliktCommand(help="Create BED file of reference ranges from
 
     val referenceFile by option(help = "Full path to the reference fasta file for filling in the intergenic/interCDS regions.  If supplied a Bed region will be created between the last GFF region and the end of the chromosome. If not, this region will be left off..")
         .default("")
+        .validate {
+            require(it.isNotBlank()) {
```

Comment:
This can be blank.  If Intergenic regions are not requested to be filled in there is no need for the reference file.

Reply:
Actually scratch that, If we pad, we could pad out past chromosome length.